java/src/test/java/vw/learner/VWFloatArrayLearnerTest.java

```diff
@@ -0,0 +1,73 @@
+package vw.learner;
```

Comment:
Basically `testLdaPredict` trains a model (+persists it), loads the model and then uses the model to get a prediction.
`testLdaPredict` tests all of these tasks. The rationale for having separate tests for each of the tasks was to make troubleshooting easier in case one of them fails.
For example if all of the 3 tests would fail then it should be straightforward to determine that the root cause is `testLdaTrainModel` failing.